docs/decisions/0002-deploy-pipelines.md

```diff
@@ -0,0 +1,53 @@
+---
+status: accepted
+date: 2022-11-04
+---
+# Deploy pipeline along with the operator
+
+## Context and Problem Statement
+
+CNBi feature include OpenShift Pipelines, that are used for task actions.<br>
+The pipelines are not entirely dependent on the CNBi Operator. <br>
+However the Operator trigger the pipelines execution, so there is a dependency 
+from Operator end.
+
+## Decision Drivers
+
+* Consistency in deployment
+* Minimize the resource location
+
+## Considered Options
+
+* Deploy the OpenShift pipelines using helm charts, and maintained on different repository.
+* Have the Operator deploy the pipeline using the tarball of the pipeline repository.
+* Include the pipelines with Operator manifests.
+
+## Decision Outcome
+
+Chosen option: "Include the pipelines with Operator manifets", because it would provide consistency on deployment.
+
+
+### Positive Consequences 
+
+* Inclusion of pipelines with operator would allow one packaging and referencing would be easier. 
+
+### Negative Consequences
+
+* As pipelines doesnt need to have dependency, this would create one.
+
+## Pros and Cons of the Options
+
+### Deploy the OpenShift pipelines using helm charts
+
+* Good, because it allows developers to work without having additional dependency.
+* Bad, because the helm chart is yet to be accepted via OpenDatahub deployments.
+* Bad, because it requires versioning of helm chart and keeping track of what getting deployed with respect to operator version. 
```

Comment:
```suggestion
* Bad, because it requires versioning of helm chart and keeping track of what getting deployed with respect to  the Operator version. 
```

Reply:
thanks :+1: